src/app/game.ts

```diff
@@ -107,6 +107,14 @@ export class Game {
     this.villageUnits.push(unit)
   }
 
+  possibleToAddUnit() {
+    return this.population > this.villageUnits.length
+  }
+
+  reduceGoldAfterRecruitment() {
```

Comment:
This is wrong. There are already prices for building unit so it must be taken from there and all resources must be reduced.

Reply:
we can have additional units that will cost more or less then 2 so it should be added dynamically

Reply:
![image](https://user-images.githubusercontent.com/38506213/106730093-7eed7a80-660e-11eb-9466-783349d31925.png)

figure out what chould be added to Knight class and how should reduceGoldAfterRecruitment method be changed